release.json

```diff
@@ -76,7 +76,7 @@
         },
         {
             "name": "gravitee-policy-cache",
-            "version": "1.13.1",
+            "version": "1.13.2-SNAPSHOT",
```

Comment:
There should be no `SNAPSHOT` as a new version will be released automatically when the PR will be merged on  `gravitee-policy-cache`